v1.1.yml

```diff
@@ -1,6 +1,6 @@
 swagger: '2.0'
 info:
-  title: clever-api
+  title: Data API
```

Comment:
As we discussed we can keep the client in its own swagger yml to address this and related issues: https://github.com/Clever/swagger-api/pull/5